net/src/packet/mod.rs

```diff
@@ -329,6 +329,11 @@ impl<Buf: PacketBufferMut> Packet<Buf> {
         &self.headers
     }
 
+    /// Get a mutable reference to the headers of this `Packet`
+    pub fn get_headers_mut(&mut self) -> &mut Headers {
+        &mut self.headers
+    }
+
```

Comment:
Is there a difference with `headers_mut` that is implemented as part of `impl<Buf: PacketBufferMut> TryHeadersMut for Packet<Buf>` in the same file?

Reply:
Second.